docs/blog/newsletter-april-2019.rst

```diff
@@ -0,0 +1,114 @@
+
+Hello folks! Beth and the newsletter team here, welcoming you to the April edition of the newsletter.
+
+Spring is here and that means conference time is speeding towards us! Portland (19-21 May) is *very nearly* sold out; I'm excited for those of you who've got tickets, the lineup looks fab. There's tons of things to tell you about the conference and I'm not even going to try to cover them all - check out the `blog post </conf/portland/2019/news/events-activities/>`_ for details on childcare, diversity tickets, shirts, and much more. 
+
+In Europe, ticket sales and CFPs are open for both Prague and the brand new Vilnius conference. The `Vilnius CFP </conf/vilnius/2019/cfp/>`_ closes **22 April**, and the `Prague CFP </conf/prague/2019/cfp/>`_ closes **30 April**. We can't wait to hear your ideas! 
+
+So that's the news this month. Read on for stories from Slack, plenty of job listings, and tons of meetups around the globe!
+
+-------------------------------
+Page length and user preference
+-------------------------------
+
+This month, the WTD community discussed whether users prefer shorter or longer pages in documentation. Both options have upsides and downsides, but most documentarians seemed to feel that longer pages work better for users; and some reported their user feedback has been strongly in favor of longer pages. Here are some of the issues folks discussed:
+
+* It's important to keep navigability at the front of your mind when you're writing longer pages. Site search, floating menus with links to sub-topics, and clear sub-headings all help make content more discoverable and skimmable. Plus, if all else fails, users can search for key terms using ``Cmd/Ctrl+F``.
+* With shorter pages, users may have to click around through more topics to find the information they need. It's also more time-consuming for users to use ``Cmd/Ctrl+F`` repeatedly on shorter pages.
+* On longer pages, it's easier for users to miss or skip over information they *need* to know before they get to what they *want* to know. One person is experimenting with adding validation checks in procedures to help minimize this problem. Most agreed that "we tried to warn you..." is not an ideal solution!
+* Because so many topics are listed on a single page, longer pages can make it more difficult to figure out which topics users are reading the most.
+
+If you want to dig in further, folks also recommended the *I'd Rather Be Writing* posts `Making Help Content Enjoyable to Read - Impossible Quest? <https://idratherbewriting.com/2011/01/25/making-help-content-enjoyable-to-read-impossible-quest-or-achievable-reality/>`_ and `Subheadings: Perhaps the Most Useful Technique in Technical Writing <https://idratherbewriting.com/2013/08/23/subheadings-perhaps-the-most-useful-technique-in-technical-writing/>`_.
+
+-----------------------
+Google's Season of Docs
+-----------------------
+
+We're excited to share with you a new event from Google: "Season of Docs"!
+
```

Comment:
Should `Season of Docs` be in quotation marks?

Reply:
Agreed.